### ISSUE TITLE
[CAIA Intake] Memorials Self Service: Pre-need address validation

### ISSUE THREAD

**Issue by barron-charles** [department-of-veterans-affairs/va.gov-team#85542]
# Content, accessibility, information architecture (CAIA) new initiative collaboration request

Use this ticket to request collaboration on a new initiative with the sitewide content, accessibility, and information architecture (CAIA) team.

## About your team

- **Team name**: `Memorials Self-Service`
- **OCTO product owner:** `Matt Self`
- **Product name:** `Apply for pre- need eligibility determination`
- **Product manager:**  `Catherine Hughes`
- **Slack channel:** [#sitewide-content-accessibility-ia](https://join.slack.com/share/enQtNzI1MjcxNzM4MDIxMi0zOWVhNjIyMWVhNDZlMzkzNmMzY2E2MzM1ZDMzYjViYmQ3YzU4OTE3ZTZhMjkzNDI4OTc2MzM3NTJhMWE0NWY2)
- **Dedicated content writer on your team (if you have one):** `N/A, UX Team`
- **Dedicated a11y specialist on your team (if you have one):** `N/A, UX Team`
 
### Which of these descriptions best fits the work we’ll partner on?
Select all that apply.
- [ ] Digitizing a new form
- [ ] Creating a new digital tool
- [x] Updating an existing form or tool
- [ ] Translating a form or tool into Spanish
- [ ] Adding new unauthenticated content to VA.gov
- [x] Updating existing unauthenticated content on VA.gov
- [ ] Something else:  `insert description`

### What's the nature of your initiative and desired outcomes?

`Tell us briefly about your work and the outcomes you’re aiming to achieve.`

`Our form, Apply for pre- need eligibility determination, is working through solutions for address validation. We hope to have the content team review our mockups and provide any content recommendations as needed. The team can comment directly in the Figma file.`

`**Note:** There are a couple further edits that could impact the content we will make early next week.`

## Collaboration timeframe

**Note:** We work on nearly every OCTO product and manage all unauthenticated content on VA.gov, so we will need to prioritize intake requests based on overall workload and VA and OCTO priorities.

### Is this work tied to a Congressional mandate, change in law or policy, or upcoming event with a specific deadline?
- [ ] Yes
- [x] No
- [ ] I’m not sure

### Where are you at in your timeline?
Tell us briefly about what you're working on now (such as initial discovery, wireframing, or usability research planning) and add any known dates for upcoming milestones or deadlines.

`Discovery, wireframing`

### Will you release this new product incrementally (for example, release to 25% of users to start)?

- [ ]  Yes
- [x]  No
 
 *Note: **If you check yes, we’ll reach out to discuss details about the content in the react widget. We use these widgets to display entry points for new products to a certain percentage of users who visit our static pages. Please refer to this [GitHub reference](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsa/teams/sitewide-content/processes/dynamic-content-process.md) for dynamic content.
 
- [x]  I acknowledge that if I change the launch to be incremental, or change it from incremental to 100%, I must notify CAIA of this change as there is additional work involved that may impact deadlines.

### Which phases of the collaboration cycle have you completed?
Select all that apply.
- [x] Design intent 
- [x] Midpoint review 
- [ ] Staging 
- [ ] None. This initiative isn’t going through the collaboration cycle.

### Collaboration cycle ticket 

If you’re going through the collaboration cycle, provide your ticket number and link:
- **Ticket Number:** [`70629`](https://github.com/department-of-veterans-affairs/va.gov-team/issues/70628)

## Supporting artifacts
Provide links to any supporting artifacts that can help us better understand your initiative and begin collaboration. Include artifacts like your product outline, user flows, mockups and prototypes, or any draft content.
- [Supporting artifact <Finalized design prototype or mockup>](https://www.figma.com/proto/6ZBtsZJt7Llx1jA5rR8T83/Pre-Need---Address-Validation-Prototype?page-id=94%3A75077&node-id=785-139239&viewport=1426%2C3089%2C0.18&t=adfuaeHxgWbKKAK6-1&scaling=min-zoom&starting-point-node-id=785%3A139239&show-proto-sidebar=1) `Finalized design prototype or mockup`
- [Supporting artifact <Research plan>](https://github.com/department-of-veterans-affairs/va.gov-team/blob/fbd4dc6ee0a4193347f3f490050e68b83f47ca8b/products/burials-memorials/pre-need/research/address-validation-research-april24/address-validation-research-plan-april24.md) `Research plan`

## Next steps
- [x] Post a link to this issue in the [#sitewide-content-sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-accessibility-ia) slack channel and tag `@Naomi Eke`, and we’ll then acknowledge receipt. We'll then review the artifacts in more detail and work with you to determine timing for collaboration. Depending on the work, we may schedule a kickoff meeting to better understand how we'll partner together.
- [ ] Review the remaining next steps in the **VFS team tasks** section here.
- [ ] **If you also need engineering support from the public websites team**, fill out their intake request form as well: [Open a public websites intake request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E)
- [ ] **If you're requesting a page/URL redirect, a URL change, or a vanity URL**, submit an additional request for this work: [Open a redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=jennymayoco%2C+kristinoletmuskat%2C+strelichl%2C+FranECross&labels=sitewide+CAIA%2C+Sitewide+IA%2C+Public+Websites%2C+VA.gov+frontend%2C+Redirect+request&projects=&template=redirect-request.md&title=Redirect+Request)
- [ ] **If you're creating an experimental design**, also contact the design system team: [Read more about experimental designs](https://design.va.gov/about/contributing-to-the-design-system/experimental-components-and-patterns#what-is-an-experimental-component-or-pattern?) <br> [Suggest an addition or update to the design system team](https://design.va.gov/about/contributing-to-the-design-system/suggest-an-addition-or-update)
- [ ] **If you're requesting a11y support for usability research**, submit a research support request next: [Open an a11y research support request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name)
- [ ] **If you are requesting content support**, please share your content source of truth (preferably as a SharePoint document).

```[tasklist]
### Content tasks
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/86234
```

**Comment by barron-charles** [comment]
Hi @aprocik1,

I've got the responses to your questions: 
- In Figma, I just added you with editor access.
- If we could do it as soon as possible, it would be much appreciated. Do you all have availability within the next couple sprints? Our team will start creating epics next sprint and we'll implement the recommendations.
- This is going to be new functionality not yet built into our form, so no staging review yet. We are not planning on conducting usability testing at this time. 
- Recording link: https://oddball-io.zoom.us/rec/share/62nmSPAllwGNbgezuB7Ri5rmqEzMb2agXpD5h_QbDGVPNJXMM0HlBJLmK5ZhkVi1.M6LBjNVhM1O7J0W8
Passcode: *1KWz#Gk

**Comment by aprocik1** [comment]
@barron-charles, thanks for your response. My goal is to provide all my content recommendations by EOD Friday 6/28. I plan on reviewing this in depth starting this week, so I may follow up with more questions. I'll post my questions and updates over the next two weeks in this intake ticket. Let me know if we need to adjust that plan.

**Comment by barron-charles** [comment]
@aprocik1 Awesome! We truly appreciate that and look forward to reviewing your recommendations.

**Comment by aprocik1** [comment]
@barron-charles, I reviewed the present / preview view of your Figma file today. And I reviewed the relevant address validation screens in the canvas / editor view. I had a hard time matching the screens across your files, so I created this [SharePoint document](https://dvagov.sharepoint.com/:w:/s/SitewideCAIA/EemmigYlPvlMjKFOhMnG4fYBp6nc1aXkC3tOROHcLPJ7ZQ?e=AKcSvR) to record my recommendations. 

I left you an outline of my logic for the address validation flow on page 1 of the document. And I left you some questions in comments on pages 15-18. After we align on the logic and pattern for one address validation flow, I'll update all the others to match. There are very subtle differences in the content for each flow depending on who's submitting the application and if they have a sponsor.

Let me know if you want to meet to discuss this. Thanks!

**Comment by barron-charles** [comment]
Hi @aprocik1 thank you for getting in and adding your recommendations. We have this [ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/85541) open for CAIA and Medallions. This was just for Pre-need. Regardless, this is helpful as we are looking into it. We'll look into the comments and address questions as needed. Thanks again.

**Comment by barron-charles** [comment]
@aprocik1 I believe we're good on our end with this ticket. Appreciate the help again.

**Comment by aprocik1** [comment]
Wonderful! I'm closing this ticket as complete.
